File: javascript/src/issue/Item.js
Item = function(name, sellIn,quality) {

    this.print =  function(){
        return name + ", " + sellIn + ", " + quality;
    };

    this.getName = function(){
        return name;
    };

    this.getSellIn = function(){
        return sellIn;
    };

    this.setSellIn = function(newSellIn){
        sellIn = newSellIn;
    };

    this.getQuality = function(){
        return quality;
    };

    this.setQuality = function(newQuality){
        quality = newQuality;
    };

};

module.exports = Item;

File: java/issue/src/issue/Item.java
package issue;

public class Item {

    private String name;

    private int sellIn;

    private int quality;

    Item(String name, int sellIn, int quality) {
        this.name = name;
        this.sellIn = sellIn;
        this.quality = quality;
    }

    @Override
    public String toString() {
        return this.name + ", " + this.sellIn + ", " + this.quality;
    }

    String getName() {
        return name;
    }

    int getSellIn() {
        return sellIn;
    }

    void setSellIn(int sellIn) {
        this.sellIn = sellIn;
    }

    int getQuality() {
        return quality;
    }

    void setQuality(int quality) {
        this.quality = quality;
    }
}